futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -415,15 +419,28 @@ class LearnerEnrollmentSerializer(
 ):  # pylint: disable=too-many-ancestors
     """Serializer for learner enrollments"""
     course_id = serializers.SerializerMethodField()
+    nafath_id = SerializerOptionalMethodField(field_tags=['nafath_id', 'csv_export'])
```

Comment:
please move the field and its logic to `LearnerBasicDetailsSerializer`. It's more convenient to have it there in my opinion. What do you think?

Reply:
as discussed in our meeting, please keep it as it is. Thank you!